src/features/campaigns/components/ActivityList/items/ActivityListItem.tsx

```diff
@@ -84,6 +86,7 @@ const ActivityListItem = ({
   href,
   color,
   meta,
+  onEventItemClick,
```

Comment:
I would give this a more neutral name, since this could be used for any activity, not just events, right?